folium/map.py

```diff
@@ -72,13 +72,14 @@ class LegacyMap(MacroElement):
         * EPSG3857 : The most common CRS for online maps, used by almost all
         free and commercial tile providers. Uses Spherical Mercator projection.
         Set in by default in Map's crs option.
-        * EPSG4326 : A common CRS among GIS enthusiasts. Uses simple Equirectangular
-        projection.
-        * EPSG3395 : Rarely used by some commercial tile providers. Uses Elliptical
-        Mercator projection.
-        * Simple : A simple CRS that maps longitude and latitude into x and y directly.
-        May be used for maps of flat surfaces (e.g. game maps). Note that the y axis
-        should still be inverted (going from bottom to top).
+        * EPSG4326 : A common CRS among GIS enthusiasts. Uses simple
+        Equirectangular projection.
+        * EPSG3395 : Rarely used by some commercial tile providers. Uses
+        Elliptical Mercator projection.
+        * Simple : A simple CRS that maps longitude and latitude into x and y
+        directly. May be used for maps of flat surfaces (e.g. game maps).
+        Note that the y axis should still be inverted
+        (going from bottom to top).
```

Comment:
You don't need to bother about that pep8 things ; we accept up to 100 chars per line.